### DISCUSSION TITLE
Veins version for omnet++ 6.0.2

### DISCUSSION THREAD

**Discussion by saidounerahma** [omnetpp/omnetpp#1150] [Q&A] [answered]
Good morning, I trust this message finds you well. 

I am writing to inquire about the compatibility of **Veins** with the **OMNeT++ 6.0.2** version, which I have recently downloaded. 

I would be grateful if you could provide me with information regarding the appropriate version of Veins that would work seamlessly with this specific version of OMNeT++.

Thank you in advance for your assistance.

Best regards, 
**R.Saïdoune**

**Comment by rhornig** [comment, marked as answer]
As far as I see:
```
$ opp_env info veins-latest

veins-5.2: The open source vehicular network simulation framework.

Requires:
- omnetpp: 6.0.3 / 6.0.2 / 6.0.1 / 6.0.0 / 6.0.x / 5.7.1 / 5.7.0 / 5.7.x / 5.6.3 / 5.6.2 / 5.6.1 / 5.6.0 / 5.6.x / 5.5.2 / 5.5.1 / 5.5.0 / 5.5.x / 5.4.2 / 5.4.1 / 5.4.0 / 5.4.x / 5.3.1 / 5.3.0 / 5.3.x
- inet: 4.2.8 / 4.2.5 / 4.2.4 / 4.2.3 / 4.2.2 / 4.2.1 / 4.2.0 / 3.8.1 / 3.7.1 / 3.7.0 / 3.6.8 / 3.6.7 / 3.6.6 / 3.6.5

Nix:
- packages: sumo

Available options:
- from-source-archive: Install from source archive on GitHub
- from-git: Install from git repo on GitHub

Note: Specify `--raw` to `opp_env info` for more details.
Note: Options can be selected by adding `--options <optionname>` to the opp_env command line, see help. Options active by default are marked with '*'.
```

so veins 5.2 does work with omnetpp 6.0.3

**Reply by rhornig** [reply]
Just install `opp_env` and then `opp_env shell --install veins-latest` and the just wait until everything is installed.

**Reply by saidounerahma** [reply]
Thanks for your answer **!******
